charon-ml/src/LlbcOfJson.ml

```diff
@@ -181,7 +181,13 @@ let crate_of_json (js : json) : (crate, string) result =
      * of function declarations: the (fresh) ids we use for those bodies
      * are simply given by: [num_functions + global_id] *)
```

Comment:
I think this comment is not valid anymore? In particular, function ids are not continuous anymore which is why we need to take the max, right?

Reply:
Oh yeah, indeed

Reply:
Fixed in https://github.com/AeneasVerif/charon/pull/344